app/views/tickets/_ticket.html.haml

```diff
@@ -8,8 +8,10 @@
         - unless ticket.description.blank?
           = markdown(ticket.description)
   %td.col-sm-1.col-md-1
-    = text_field_tag("tickets[][#{ticket.id}]", 0,
-      type: 'number', min: 0, class: "form-control quantity", 'data-id' => ticket.id)
+    - options = { type: 'number', min: 0, class: "form-control quantity", 'data-id' => ticket.id }
+    - if ticket.registration_ticket?
```

Comment:
:art:  you could write this two lines together: 

`- options[:max] = 1 if ticket.registration_ticket?`

Reply:
Added some more conditions in this if condition therefore going back to previous implementation.